static/src/js/components/ProjectCollections/skeleton.jsx

```diff
@@ -51,22 +51,25 @@ export const projectHeader = [
     top: 6,
     height: 12,
     width: 80,
-    radius: 2
+    radius: 2,
+    'data-testid': 'skeleton-item'
```

Comment:
Are these being used?

Reply:
Looks like ProjectHeader line 78 uses this.